iib/workers/tasks/build_merge_index_image.py

```diff
@@ -415,6 +424,12 @@ def is_bundle_version_valid(bundle_path: str, valid_ocp_version: str) -> bool:
         raise IIBError(f'Invalid OCP version, "{valid_ocp_version}", specified in Index Image')
     try:
         bundle_version_label = get_image_label(bundle_path, 'com.redhat.openshift.versions')
+        if allow_no_ocp_version and not bundle_version_label:
+            log.debug(
+                'Marking bundle %s without label `com.redhat.openshift.versions` as valid.',
+                bundle_path,
+            )
```

Comment:
What about using it as `logging.warning` instead? Then we may see in the logs when actually processing these images without need to fetch the current config.

Also if the config changes we may preserve which images were processed without the versions.

Reply:
I am open to change it to info. But not warning. It is not an error state. It is expected and I just wanted to have any information in logfile about processing bundles without the label.

Reply:
Sure, makes sense, `info` looks good to me

Reply:
changed

Reply:
Thank you!